eng/pipelines/coreclr/update-checkedjit-map.yml

```diff
@@ -0,0 +1,32 @@
+trigger:
+  batch: false
+  branches:
+    include:
+    - main
+    - release/*.*
+  paths:
+    include:
+    - '*'
+
+# This pipeline is supposed to be run only on merged changes
+# and should not be triggerable from a PR. 
+pr: none
+
+jobs:
+
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/coreclr/templates/update-checkedjit-map-job.yml
+    buildConfig: checked
+    platforms:
+    - OSX_arm64
+    - OSX_x64
+    # Currently, Linux arm/arm64 machines don't have the Python 'pip3' tool, nor the azure-storage-blob package that
+    # is required to do the JIT upload to Azure Storage. Thus, these platforms are disabled. If we can figure out how
+    # to get Python properly configured, then re-enable them.
+    # - Linux_arm
+    # - Linux_arm64
+    - Linux_x64
+    - windows_x64
+    - windows_x86
+    - windows_arm64
```

Comment:
Why do we want to run this on all possible configurations? One `windows_x64` machine is good enough to update the git hash, right?

Reply:
Also, can we do this as part of `jitrollingbuild` pipeline itself?